src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const { slug } = useParams();
+
+  const mother = people.find(possibleMother => (
+    possibleMother.name === person.motherName
+  )) || null;
+  const father = people.find(possibleFather => (
+    possibleFather.name === person.fatherName
+  )) || null;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': person.slug === slug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${person.slug}`}
+          className={cn({
+            'has-text-danger': person.sex === 'f',
+          })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died}</td>
+      <td>
+        {mother && (
+          <Link to={`/people/${mother.slug}`} className="has-text-danger">
+            {mother.name}
+          </Link>
```

Comment:
Pay attention to requirements:
![image](https://github.com/mate-academy/react_people-table-basics/assets/63243136/e7a431c0-dd3d-497c-8d15-a74f2c3d4694)